frontend/src/container/QueryBuilder/filters/OperatorsSelect/OperatorsSelect.tsx

```diff
@@ -0,0 +1,32 @@
+import { Select } from 'antd';
+import React from 'react';
+// ** Types
+import { SelectOption } from 'types/common/select';
+// ** Helpers
+import { transformToUpperCase } from 'utils/transformToUpperCase';
+
+import { OperatorsSelectProps } from './OperatorsSelect.interfaces';
+
+export function OperatorsSelect({
+	operators,
+	value,
+	onChange,
+	...props
+}: OperatorsSelectProps): JSX.Element {
+	const operatorsOptions: SelectOption<string, string>[] = operators.map(
+		(operator) => ({
+			label: transformToUpperCase(operator),
+			value: operator,
+		}),
+	);
+
+	return (
+		<Select
+			options={operatorsOptions}
+			value={value}
+			onChange={onChange}
+			// eslint-disable-next-line react/jsx-props-no-spreading
+			{...props}
```

Comment:
would it be possible not to spread

Reply:
I created a reusable selector for the operators and we can pass as props any prop for native select + our props. Is the spread bad practice?

Reply:
we should be aware what props child is taking